TF-M image signing failure in Travis

From https://travis-ci.com/github/ARMmbed/mbed-os-example-psa/jobs/512116529:
```
Total Static RAM memory (data + bss): 18312(+18312) bytes
Total Flash memory (text + data): 24676(+24676) bytes

[159/159] Running utility command for mbed-post-build-bin-ARM_MUSCA_S1
FAILED: mbed-os/targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/CMakeFiles/mbed-post-build-bin-ARM_MUSCA_S1.util 
cd /home/travis/build/ARMmbed/mbed-os-example-psa/cmake_build/ARM_MUSCA_S1/release/GCC_ARM/mbed-os/targets/TARGET_ARM_SSG/TARGET_MUSCA_S1 && /home/travis/virtualenv/python3.8.3/bin/python3.8 /home/travis/build/ARMmbed/mbed-os-example-psa/mbed-os/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/generate_mbed_image.py --tfm-target musca_s1 --target-path /home/travis/build/ARMmbed/mbed-os-example-psa/mbed-os/targets/TARGET_ARM_SSG/TARGET_MUSCA_S1 --secure-bin /home/travis/build/ARMmbed/mbed-os-example-psa/mbed-os/targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/tfm_s.bin --non-secure-bin /home/travis/build/ARMmbed/mbed-os-example-psa/cmake_build/ARM_MUSCA_S1/release/GCC_ARM/mbed-os-example-psa.bin
Traceback (most recent call last):
  File "/home/travis/build/ARMmbed/mbed-os-example-psa/mbed-os/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/generate_mbed_image.py", line 197, in <module>
    sign_and_merge_tfm_bin(args.tfm_target, args.target_path, args.non_secure_bin, args.secure_bin)
  File "/home/travis/build/ARMmbed/mbed-os-example-psa/mbed-os/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/generate_mbed_image.py", line 80, in sign_and_merge_tfm_bin
    raise Exception("Unable to sign " + target_name +
Exception: Unable to sign musca_s1 secure binary, Error code: 1
ninja: build stopped: subcommand failed.
ERROR: CMake invocation failed!
More information may be available by using the command line option '-v'.
The command "mbedtools compile -t GCC_ARM -m ${TARGET_NAME} -b ${PROFILE}" exited with 1.
```
The image signing script failed, but the error is not reproducible on my local environment.

--- a/.travis.yml
+++ b/.travis.yml
@@ -71,11 +71,10 @@ matrix:
         - sudo ln -s $(which ccache) arm-none-eabi-g++
         - export PATH="/usr/lib/ccache:$PATH"
         - popd
-        # Fetch mbed-os: We use manual clone, with depth=1 and --single-branch to save time.
-        - git clone --depth=1 --single-branch https://github.com/ARMmbed/mbed-os.git;
         # Install Mbed CLI and dependencies
         - pip install --upgrade mbed-tools
-        - pip install -r mbed-os/tools/cmake/requirements.txt
+        - mbed-tools deploy
+        - pip install -r mbed-os/requirements.txt
       script:
         - echo mbedtools compile -t GCC_ARM -m ${TARGET_NAME} -b ${PROFILE}
         - mbedtools compile -t GCC_ARM -m ${TARGET_NAME} -b ${PROFILE}

